akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcClusterSpec.scala

```diff
@@ -108,18 +108,27 @@ abstract class MultiDcSpec(config: MultiDcSpecConfig)
         cluster.join(third)
       }
 
-      // should be able to join and become up since the
-      // unreachable is between dc1 and dc2,
-      within(10.seconds) {
-        awaitAssert(clusterView.members.filter(_.status == MemberStatus.Up) should have size (5))
+      runOn(third, fourth, fifth) {
+        // should be able to join and become up since the
+        // unreachable is between dc1 and dc2,
+        within(10.seconds) {
+          awaitAssert(clusterView.members.filter(_.status == MemberStatus.Up) should have size (5))
+        }
       }
 
       runOn(first) {
         testConductor.passThrough(first, third, Direction.Both).await
       }
-      runOn(first, second, third, fourth) {
-        awaitAssert(clusterView.unreachableMembers should not be empty)
+
+      // reachable again
+      awaitAssert(clusterView.unreachableMembers should be(empty))
```

Comment:
Yeah, this is more as a guard to know when to move on, I guess we will have to update these tests when that is in place.